code/dynamic_programming/src/Count_Subsequence_Having_Product_Less_Than_K/Solution.cpp

```diff
@@ -0,0 +1,67 @@
+//Given a positive array, find the number of subsequences having product smaller than K.
+// LINK - https://www.geeksforgeeks.org/count-subsequences-product-less-k/#
+#include<bits/stdc++.h>
```

Comment:
not a good practice to use such header, can you please use only the headers needed?